components/builder-worker/src/runner/mod.rs

```diff
@@ -102,20 +102,28 @@ impl Runner {
 
     pub fn run(mut self) -> Job {
         if let Some(err) = util::validate_integrations(&self.workspace).err() {
-            error!("failed to validate integrations err={:?}", err);
+            let msg = format!("Failed to validate integrations, err={:?}", err);
+            error!("{}", msg);
```

Comment:
We probably mean `debug` here, this will wake us up if somebody enters invalid credentials